dockerfiles/Dockerfile

```diff
@@ -6,11 +6,12 @@ WORKDIR /tmp
 
 COPY README.md README.md
 COPY LICENSE LICENSE
-COPY tipg/ tipg/
 COPY pyproject.toml pyproject.toml
+RUN mkdir -p tipg && echo '__version__="0.0.0"' >/tmp/tipg/__init__.py
```

Comment:
🤔 what's that for?

Reply:
To install dependencies early in the docker chain so cache doesn't change

Reply:
ok, seems like pretty `hacky` I don't think we should do that.

If creating docker image speed is really an issue we could do something like

```
RUN pip install pip-tools
...
COPY pyproject.toml pyproject.toml
RUN pip-compile -o requirements.txt pyproject.toml
RUN pip install -r requirement.txt
```

But again, we shouldn't care about the speed of building the docker image.